docs/topics/exceptions.rst

```diff
@@ -77,3 +78,25 @@ NotSupported
 
 This exception is raised to indicate an unsupported feature.
 
+StopDownload
+-------------
+
+.. versionadded:: 2.2
+
+.. exception:: StopDownload(fail=False)
```

Comment:
What do you think about either the removing the default value, or making it True by default?

Argument for removing the default: `StopDownload(fail=False)` is easier to understand, and not much longer to write.

Argument for making fail=True by default: this feature can be seen as a much more flexible DOWNLOAD_MAXSIZE, and DOWNLOAD_MAXSIZE causes errback to be called.